code/app/src/main/java/com/adobe/marketing/edge/identity/app/ui/CustomIdentityFragment.kt

```diff
@@ -93,6 +104,44 @@ class CustomIdentityFragment : Fragment() {
             Identity.removeIdentity(item, namespace)
         }
 
+        // Button for Set Ad ID behavior
+        // Sets the advertising identifier set in the corresponding textfield using the MobileCore API
+        root.findViewById<Button>(R.id.btn_set_ad_id).setOnClickListener {
+            val adId: String? = sharedViewModel.adId.value
+            MobileCore.setAdvertisingIdentifier(adId)
+        }
+
+        // For details on implementation tips and differences between Google Play Services Ads vs AndroidX Ads,
+        // please see the project Documentation -> AEPEdgeIdentity.md : Test App -> GAID Implementation
```

Comment:
please see the project Documentation -> AEPEdgeIdentity.md : Android Test App -> GAID Implementations